src/catalog/src/builtin.rs

```diff
@@ -3261,12 +3261,12 @@ pub static MZ_RELATIONS: Lazy<BuiltinView> = Lazy::new(|| {
         name: "mz_relations",
         schema: MZ_CATALOG_SCHEMA,
         oid: oid::VIEW_MZ_RELATIONS_OID,
-        column_defs: Some("id, oid, schema_id, name, type, owner_id, privileges"),
+        column_defs: Some("id, oid, schema_id, name, type, owner_id, cluster_id, privileges"),
```

Comment:
Probably need to add this to mz_catalog.td (and for the other relations too).

Reply:
I didn't see a testdrive test that cares about all the columns, but I did have to update some SLT tests.

Reply:
Erp, I meant `mz_catalog.md`, and then you'll need to run `ci/test/lint-docs-catalog.sh --rewrite`.

Reply:
Ah cool, just pushed those changes.